cinema/models.py

```diff
@@ -89,9 +85,7 @@
             (self.row, "row", "rows"),
             (self.seat, "seat", "seats_in_row"),
         ]:
-            count_attrs = getattr(
-                self.movie_session.cinema_hall, cinema_hall_attr_name
-            )
+            count_attrs = getattr(self.movie_session.cinema_hall, cinema_hall_attr_name)
             if not (1 <= ticket_attr_value <= count_attrs):
```

Comment:
The validation logic in the `clean` method is correct, but ensure that the `movie_session` attribute is properly set before calling `clean`. Otherwise, accessing `self.movie_session.cinema_hall` might result in an error if `movie_session` is not assigned.